frontend/packages/core/src/Input/toggle-button-group.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import {
+  ToggleButton as MuiToggleButton,
+  ToggleButtonGroup as MuiToggleButtonGroup,
+} from "@material-ui/lab";
+
+export interface ToggleButtonGroupProps {
+  exclusive?: boolean;
+  currentValue: string;
+  onChange: (event: React.ChangeEvent<{}>, value: string) => void;
+  toggleButtonValues: string[];
+}
+
+const StyledMuiToggleButtonGroup = styled(MuiToggleButtonGroup)({
+  ".MuiToggleButton-root": {
+    "&.Mui-selected": {
+      color: "#3548D4",
+      backgroundColor: "#3548D41F",
+    },
+  },
+  padding: "6px 16px 16px 6px",
+});
```

Comment:
Could we adjust the CSS to add some inset properties? To make it look a bit more like the button is actually pressed vs just selected. Unless this is based off of Figma already.

Reply:
The figma doesn't seem to have an inset, but here's what it looks like with 1px all around. do you think that works? or maybe it should just be vertical inset not horizontal?
![inset1](https://user-images.githubusercontent.com/66325812/168692558-f0b1ac7c-3390-4ad3-94d8-14d72507d505.gif)
?

Reply:
Yeah maybe try horizontal, them changing height is a little odd lol. But it definitely looks pressed so I think it's a step in the right direction.